utils/settings.py

```diff
@@ -9,7 +9,7 @@
 
 
 console = Console()
-config = dict  # autocomplete
+config: dict | None = None  # autocomplete
```

Comment:
Not everyone has 3.10

add at the beginning 
```python
from typing import Union
```
and use
```suggestion
config: Union[dict, None] = None  # autocomplete
```

or use, it'll work as well
```suggestion
config = dict()  # autocomplete
```

Reply:
Thank you for taking the time to look through and explain everything.